lib/puppet/parser/functions/foreman.rb

```diff
@@ -36,6 +36,7 @@
 #
 # Happy Foreman API-ing!
 
+require "pry"
```

Comment:
Looking at the original problem (https://github.com/theforeman/puppet-foreman/issues/482) I find it hard to see how increasing the timeout solves it. That makes me wonder if pry has some sort of side effect.